config/webpack.config.dev.js

```diff
@@ -1,48 +1,12 @@
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const FaviconsWebpackPlugin = require('favicons-webpack-plugin');
-const Dotenv = require('dotenv-webpack');
+const webpackMerge = require('webpack-merge');
+const common = require('./webpack.config.common');
 const path = require('path');
 
-module.exports = {
+module.exports = webpackMerge(common, {
     mode: 'development',
```

Comment:
If you write the mode here you don't need the `--mode development` in the script on package.json